### DISCUSSION TITLE
Copilot stops working

### DISCUSSION THREAD

**Discussion by JKamsker** [community/community#106390] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Manual trigger doesnt work, also no new log is created. The menu items are also grayed out. Restart usually helps but is annoying.
![grafik](https://github.com/community/community/assets/11245306/0348ae01-9055-491b-824a-01fb0f8be391)
![grafik](https://github.com/community/community/assets/11245306/dcea4100-9dad-489b-9cb4-1577f94a146c)

**Comment by ayim** [comment]
Hey! seems serious, sorry you're running into this.

1) can you confirm the Copilot version and VS version you're using?
2) The next time you run into this, could you file a bug here so we can get additional logs?

![image](https://github.com/community/community/assets/10951869/74010b89-ea26-4101-99a5-5341a2598097)

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
github: ayim | aayim@microsoft.com

**Comment by JKamsker** [comment]
@ayim It happened again: https://developercommunity.visualstudio.com/t/Copilot-stops-working-when-requesting-co/10607240?port=1025&fsid=ec243082-aade-4a89-817b-97efc4dfe3ac

**Reply by ayim** [reply]
Thanks for filing again, we'll take a look

**Comment by ValdirGiorgi** [comment]
I have a similar problem

https://github.com/microsoft/vscode-copilot-release/issues/1016

**Comment by samuelavike1** [comment]
Remove the co pilot extension and install it again.

**Reply by ValdirGiorgi** [reply]
I did it and the problem continues

**Reply by ayim** [reply]
@ValdirGiorgi you'll want to open a new discussion on this page and tag @Akash1134 so he can track the VSCode issue.
I'm tracking the Visual Studio related issue on this one.
